### PR TITLE
Point to New Docker Img

### PR DESCRIPTION
**Pull request by alan20854**
points to docker img containing circle service

### CODE REVIEW

File: .github/workflows/nightfalldlp.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: nightfallDLP action step
-        uses: nightfallai/nightfall_dlp_action@v0.0.5
+        uses: nightfallai/nightfall_dlp_action@BumpVersionCircle
```
Comment (alan20854) [inline .github/workflows/nightfalldlp.yml:16]: rm this before merging

**Comment by alan20854** [comment]
Post merge, create a new tag for 0.0.6, and make another PR updating the action's own workflow to point to new tag

File: test/finding.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+my phone number is 301-231-5824
```
Comment (alan20854) [inline test/finding.txt:1]: delete this once you confirm api platform is fixed

File: action.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: Nightfall DLP Action
 description: Scan Pull Requests for sensitive information, like credentials & secrets, PII, credit card numbers, and more.
 runs:
   using: 'docker'
-  image: 'docker://nightfallai/nightfall_cli:v0.0.3'
+  image: 'docker://nightfallai/nightfall_code_scanner:v0.0.4'
```
Comment (henkejosh) [inline action.yml:5]: what changes are in the new image? Anything that needs to be updated in the readme or release notes?

Comment (alan20854) [reply action.yml:5]: This version contains all the code for the circleci service, nothing related to the github action. I've created the release + notes in https://github.com/nightfallai/nightfall_code_scanner explaining that. 

Does this mean I don't need to update which image our Github action is pointing to, or should I still do so just to make sure we're pointing to the most up-to-date docker image?

**Comment by alan20854** [comment]
closing due to no changes in functionality for github action

**Pull request closed without merging**